### DISCUSSION TITLE
Hi. I'm having some error in the Org Level Flowviz.

### DISCUSSION THREAD

**Discussion by Karloyski** [nbrown02/FlowViz-Jira#29] [Q&A] [answered]
Hi. I hope you can help me resolve this error and point me to the path that will resolve this. I am using the Org Level FlowViz template. Below is the error:

WorkItems Bl0cked
Failed to save modifications to the server. Error returned: 'OLE DB or ODBC error: [DataSource.Error] OData: Invalid JSON. Unexpected end of input was found in JSON content. Not all object and array scopes were closed.. OLE DB or ODBC error: Exception from HRESULT: 0x80040E4E. The current operation was cancelled because another operation in the transaction failed. OLE DB or ODBC error: Exception from HRESULT: 0x80040E4E. '.
WorkItems WIP Age
Failed to save modifications to the server. Error returned: 'OLE DB or ODBC error: [DataSource.Error] OData: Invalid JSON. Unexpected end of input was found in JSON content. Not all object and array scopes were closed.. OLE DB or ODBC error: Exception from HRESULT: 0x80040E4E. The current operation was cancelled because another operation in the transaction failed. OLE DB or ODBC error: Exception from HRESULT: 0x80040E4E. '.
WorkItems FlowEfficiency
Failed to save modifications to the server. Error returned: 'OLE DB or ODBC error: [DataSource.Error] OData: Invalid JSON. Unexpected end of input was found in JSON content. Not all object and array scopes were closed.. OLE DB or ODBC error: Exception from HRESULT: 0x80040E4E. The current operation was cancelled because another operation in the transaction failed. OLE DB or ODBC error: Exception from HRESULT: 0x80040E4E. '.

**Comment by nbrown02** [comment]
Hi there,This error reads like a timeout issue. Are you trying to load a large set of data?Thanks Nick Sent from my iPhoneOn 20 Oct 2023, at 00:25, Karloyski ***@***.***> wrote:﻿
Hi. I hope you can help me resolve this error and point me to the path that will resolve this. I am using the Org Level FlowViz template. Below is the error:
WorkItems Bl0cked
Failed to save modifications to the server. Error returned: 'OLE DB or ODBC error: [DataSource.Error] OData: Invalid JSON. Unexpected end of input was found in JSON content. Not all object and array scopes were closed.. OLE DB or ODBC error: Exception from HRESULT: 0x80040E4E. The current operation was cancelled because another operation in the transaction failed. OLE DB or ODBC error: Exception from HRESULT: 0x80040E4E. '.
WorkItems WIP Age
Failed to save modifications to the server. Error returned: 'OLE DB or ODBC error: [DataSource.Error] OData: Invalid JSON. Unexpected end of input was found in JSON content. Not all object and array scopes were closed.. OLE DB or ODBC error: Exception from HRESULT: 0x80040E4E. The current operation was cancelled because another operation in the transaction failed. OLE DB or ODBC error: Exception from HRESULT: 0x80040E4E. '.
WorkItems FlowEfficiency
Failed to save modifications to the server. Error returned: 'OLE DB or ODBC error: [DataSource.Error] OData: Invalid JSON. Unexpected end of input was found in JSON content. Not all object and array scopes were closed.. OLE DB or ODBC error: Exception from HRESULT: 0x80040E4E. The current operation was cancelled because another operation in the transaction failed. OLE DB or ODBC error: Exception from HRESULT: 0x80040E4E. '.

**Reply by Karloyski** [reply]
Hi Nick,
Appreciate your prompt response. Yes I am pulling almost 9 years of data across the whole organization. How would you suggest I limit the data pull to just the past 3 years of data. Which table is the main table to make that edit from? Thank you very much!

**Comment by nbrown02** [comment]
Ah - well all the tables have a limit on bringing in only the last year’s worth of data. However some are pulling back every change to the issue so, will likely still be a large dataset if you are doing lots of projects…Where it says 52w in the queries you’d need to change this to be a shorter range, you also need to ensure you do this with the functions…Sent from my iPhoneOn 20 Oct 2023, at 02:08, Karloyski ***@***.***> wrote:﻿
Hi Nick,
Appreciate your prompt response. Yes I am pulling almost 9 years of data across the whole organization. How would you suggest I limit the data pull to just the past 3 years of data. Which table is the main table to make that edit from? Thank you very much!

**Reply by Karloyski** [reply, marked as answer]
Its working now Nick. Thank you. I had to build multiple functions and choke my project list so that I am not pulling in a very big dataset. Appreciate your help!
